src/lib/components/Cytoscape.react.js

```diff
@@ -283,6 +288,9 @@ class Cytoscape extends Component {
 
         this.cyResponsiveClass = new CyResponsive(cy);
         this.cyResponsiveClass.toggle(this.props.responsive);
+
+        this.cyCtxMenuClass = new CyCtxMenu(cy);
+        this.cyCtxMenuClass.ctxmenuUpdate(this.props);
```

Comment:
This line seems to be assigned even when the user does not necessarily want to use cyCtxMenu. Is there a way to ensure that this is only assigned whenever it is needed?